docs/docs/aztec/concepts/accounts/keys.md

```diff
@@ -3,157 +3,123 @@ title: Keys
-This means that the keys used for the same user in two different application contracts will be different and potential leak of the scoped keys would only affect 1 application.
+### Address keys
 
-This also allows per-application auditability.
-A user may choose to disclose their incoming and outgoing viewing keys for a given application to an auditor or regulator (or for 3rd party interfaces, e.g. giving access to a block explorer to display my activity), as a means to reveal all their activity within that context, while retaining privacy across all other applications in the network.
+Address keys are used for account address derivation. 
 
-In the case of nullifier keys, there is also a security reason involved.
-Since the nullifier secret is exposed to the application contract to be used in the nullifier computation, the contract may accidentally or maliciously leak it.
-If that happens, only the nullifier secret for that application is compromised (`nsk_app` and not `nsk_m`).
+Address keys are a pair of keys `AddressPublicKey` and `address_sk` where `address_sk` is a scalar defined as `address_sk = pre_address + ivsk` and `AddressPublicKey` is an elliptic curve point. `pre_address` can be thought of as a hash of all account’s key pairs and functions in the account contract: `pre_address := poseidon2(public_keys_hash, partial_address)` where `partial_address := poseidon2(contract_class_id, salted_initialization_hash)` and `public_keys_hash := poseidon2(Npk_m, Ivpk_m, Ovpk_m, Tpk_m)`.
 
-Above we mentioned that the notes typically contain `Npk_m`.
-It might seem like a mistake given that the notes are nullified with `nsk_app`.
-This is intentional and instead of directly trying to derive `Npk_m` from `nsk_app` we instead verify that both of the keys were derived from the same `nsk_m` in our protocol circuits.
-
-## Protocol key types
-
-All the keys below are Grumpkin keys (public keys derived on the Grumpkin curve).
+:::note
+Under the current design Aztec protocol does not use `Ovpk` (outgoing viewing key) and `Tpk` (tagging key). However, formally they still exist and can be used by developers for some non-trivial design choices if needed. 
+:::
 
-## Nullifier keys
+### Signing keys
 
-Whenever a note is consumed, a nullifier deterministically derived from it is emitted.
-This mechanisms prevents double-spends, since nullifiers are checked by the protocol to be unique.
-Now, in order to preserve privacy, a third party should not be able to link a note hash to its nullifier - this link is enforced by the note implementation.
-Therefore, calculating the nullifier for a note requires a secret from its owner.
+Thanks to the native account abstraction, authorization logic can be implemented in an alternative way using Google authorization credentials, vanilla password logic or Face ID mechanism. In all these cases signing keys are not relevant.
```

Comment:
link to the concept docs that talk about account abstraction on Aztec

and also backlink from that page to here, since we should talk about signing keys there too!